test/local/routes/recovery_email_status.js

```diff
@@ -0,0 +1,102 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+require('ass')
+
+var sinon = require('sinon')
+
+var test = require('../../ptaptest')
+var mocks = require('../../mocks')
+var config = require('../../../config').getProperties()
+
+var P = require('../../../lib/promise')
+var error = require('../../../lib/error')
+
+function getMockedRoute(path, mockDB) {
+  var routes = require('../../../lib/routes/account')(
+    mocks.mockLog(),
+    require('crypto'),
+    P,
+    undefined,
+    require('joi'),
+    require('../../../lib/error'),
+    mockDB,
+    undefined,
+    undefined,
+    config,
+    undefined,
+    undefined,
+    undefined
+  )
+  for (var i = 0; i < routes.length; i++) {
+    if (routes[i].path === path) {
+      return routes[i]
+    }
+  }
+  throw new Error('Could not find route: ' + path)
+}
```

Comment:
Sure, if you're working on something similar, it makes sense to standardize on approach - do you have a WIP of what you were doing with the push ones?